src/components/proposals/brand/navbar/navbar.scss

```diff
@@ -0,0 +1,466 @@
+@import "../../../../assets/scss/variables";
+@import "../../../../assets/scss/utilities";
+
+.iowa-bar--narrow .logo--tab {
+  margin-right: 0px;
+}
+
+.links{
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+}
+
+.top-nav {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content:space-between
+}
+
+.top{
+  display: flex;
+  flex-direction: row;
+}
+
+.site-name-wrapper{
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  width:100%;
+  padding-left:0px;
+
+  h1 >a {
+    display: flex;
+    align-items: center;
+    margin: 0 auto 0 0;
+    text-align: left;
+    font-size: 1.5rem;
+    font-weight: 300;
+    line-height: 1;
+    font-family: "Roboto", sans-serif;
+    letter-spacing: -0.02em;
+  }
+}
+
+.toggle {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding:6px 6px;
+
+  background-color: $primary;
+  a {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+    height: 100%;
+    color: $secondary;
+    text-align: center;
+    text-decoration: none;
+    font-size: 17px;
+    width: 40px;
+    height: 40px;
+  }
+  a:hover {
+    background-color: $white;
+    color: $secondary;
+  }
+}
+
+
+.site-links {
+  display: flex;
+  flex-wrap:wrap;
+  position: relative;
+
+  a {
+    float: none;
+    display: none;
+  }
+
+  .dropdown {
+    float: none;
+    display: none;
+  }
+}
+
+.dropdown {
+  float: left;
+  position: relative;
+}
+
+.dropdown-content {
+  display: none;
+  position: relative;
+
+  a {
+    display: block;
+  }
+
+  .open a {
+    display: block;
+  }
+}
+
+.icons{
+  padding-left:5px ;
+}
+
+.invisible{
+
+  display: none
+}
+
+//mobile
+@media screen and (max-width: 855px) {
+
+
+  .site-links.responsive{
+    padding:10px;
+    background-color: white;
+  }
+
+  .site-links.responsive a,
+  .site-links.responsive .dropdown {
+    float: none;
+    display: block;
+
+  }
+
+  .site-links.responsive {
+    position: absolute;
+    width: 100%;
+    box-shadow: 0 5px 10px -2px rgba(0, 0, 0, 0.26);
+    -webkit-box-shadow: 0 5px 10px -2px rgba(0, 0, 0, 0.26);
+  }
+
+  .site-links.responsive a {
+    color: $secondary;
+    text-decoration: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0 10px;
+    align-items: center;
+   // margin: 0 auto;
+    text-align: center;
+    height: 50px;
+    width:100%;
+    font-size: 1.1375rem;
+
+    background-color: white;
+    border-bottom: 1px solid rgba(0,0,0,0.2);
+    border-top: 1px solid rgba(255,255,255,0.05);
+  }
+
+  .dropdown .dropdown-content{
+
+    a{
+      color: #666;
+      font-weight: 400;
```

Comment:
Change font-weight to `font-weight: 600;` to match Superfish styles?  `400` is fine for child links.